### ISSUE TITLE
[quest=10198] Information Gathering

### ISSUE THREAD

**Issue by pangolp** [azerothcore/azerothcore-wotlk#16519]
### Current Behaviour

a

### Expected Blizzlike Behaviour

a

### Steps to reproduce the problem

a

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/590dcca82b9c6e5d5f49fb5f954b084a88070f84

### Operating system

Ubuntu 20.04

**Comment by Gultask** [comment]
@pangolp Could you add some coordinates or .tele commands that we can use for quick reference? ty for the report

**Comment by pangolp** [comment]
> @pangolp Could you add some coordinates or .tele commands that we can use for quick reference? ty for the report

Yes, I wanted to. But I didn't find the npc and then I forgot. Later, I edit the report and add the coordinates.

**Comment by pangolp** [comment]
I have added additional information about the npc.
Although it seems that the mission is part of a chain.
Or it requires a certain reputation to be taken.
It is not enough just to have the level.
